scripts/validate_docstrings.py

```diff
@@ -233,9 +234,19 @@ def correct_parameters(self):
     def parameter_type(self, param):
         return self.doc_parameters[param][0]
 
-    def parameter_desc(self, param):
+    def raw_parameter_desc(self, param):
```

Comment:
@datapythonista I know you wanted this but I would prefer not to include unless it serves a purpose. May even be better served as a keyword argument going forward instead of a dedicated method so would rather not go down this path until needed.

Outside of that I think this change looks good